### ISSUE TITLE
Biospec - Dshbrd - Add PWA login mode to Contact Information 

### ISSUE THREAD

**Issue by wyattks** [episphere/connect#317]
On the Research Check-in screen, include the PWA Login Method in the Contact Information.  i.e. the email they used to set up their account.  Should automatically update if participant updates.
See attachment

[Contact Information Add PWA login mode.docx](https://github.com/episphere/connect/files/6346344/Contact.Information.Add.PWA.login.mode.docx)

**Comment by anthonypetersen** [comment]
@brotzmanmj has there been any more conversation on where the Login Mode should be displayed? At this point in dev we are hiding Contact Information as well as the Verify button.

**Comment by brotzmanmj** [comment]
@anthonypetersen Hi Tony, it should be displayed somewhere on the Check In/Out Screen. Perhaps toward the top of the page.

**Comment by anthonypetersen** [comment]
Similar to #342 this change has been merged into the branch Iqbal has made for his navigational changes. Once that branch is merged into dev these changes will be visible.
